scipy/special/_add_newdocs.py

```diff
@@ -201,6 +201,61 @@ def add_newdoc(name, doc):
     ----------
     .. [1] https://en.wikipedia.org/wiki/Voigt_profile
 
+    Examples
+    --------
+    Calculate the function at point 2 for ``sigma=1`` and ``gamma=1``.
+
+    >>> from scipy.special import voigt_profile
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> voigt_profile(2, 1., 1.)
+    0.09071519942627544
+
+    Calculate the function at several points by providing a NumPy array
+    for `x`.
+
+    >>> values = np.array([-2., 0., 5])
+    >>> voigt_profile(values, 1., 1.)
+    array([0.0907152 , 0.20870928, 0.01388492])
+
+    Plot the function for different parameter sets.
+
+    >>> (fig, ax) = plt.subplots(figsize=(8, 8))
```

Comment:
Nitpick: remove extraneous parentheses:
```suggestion
    >>> fig, ax = plt.subplots(figsize=(8, 8))
```